server.js

```diff
@@ -1,35 +1,107 @@
-import express from "express";
-import cors from "cors";
-import mongoose from "mongoose";
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
-mongoose.connect(mongoUrl);
-mongoose.Promise = Promise;
-
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
-const port = process.env.PORT || 8080;
-const app = express();
-
-// Add middlewares to enable cors and json body parsing
-app.use(cors());
-app.use(express.json());
-
-// Start defining your routes here
+import express from "express"
+import expressListEndpoints from "express-list-endpoints"
+import cors from "cors"
+import mongoose from "mongoose"
+import netflixData from "./data/netflix-titles.json"
+import dotenv from "dotenv"
+
+dotenv.config()
+
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
+mongoose
+  .connect(mongoUrl)
+  .then(() => console.log("Connected to MongoDB"))
+  .catch((error) => console.error("Error connecting to MongoDB:", error))
+
+mongoose.Promise = Promise
+
+// Set Schema
+const { Schema } = mongoose
+
+const netflixSchema = new Schema({
+  show_id: Number,
+  title: String,
+  director: String,
+  cast: String,
+  country: String,
+  date_added: Date,
+  release_year: Number,
+  rating: String,
+  duration: String,
+  listed_in: String,
+  description: String,
+  type: String,
+})
+
+const NetflixModel = mongoose.model("Netflix", netflixSchema)
+
+// Set Seed
+if (process.env.RESET_DATABASE) {
```

Comment:
did you add the env to the hosting service settings? (Render) The DB is empty at the moment

Reply:
Hi Antonella,
I checked again and fixed the bug. I saw, that I wrote Netflix instead of NetflixModel. 
It should work now. Thank you :)